cycledash/static/js/comments/components/comment.js

```diff
@@ -23,11 +23,17 @@ var LatestComments = React.createClass({
     comments: React.PropTypes.arrayOf(React.PropTypes.object).isRequired
   },
   render: function() {
+    var numComments = this.props.comments.length;
+    var recentComments;
+    if (numComments > 0) {
+      recentComments =  <div><h4>Last {numComments} Comments<a href='/comments' className='all-comments'>See all</a></h4>
```

Comment:
Nits: there should be one space after the =, and the JSX should be formatted like HTML (e.g. nesting and new lines).